snapcraft/tests/unit/store/test_store_client.py

```diff
@@ -1432,6 +1433,16 @@ def test_conflicting_force(self):
         result = self.client.push_metadata('basic', metadata, True)
         self.assertIsNone(result)
 
+    def test_snap_not_found(self):
+        self._setup_snap()
+        metadata = {'field_ok': 'foo'}
```

Comment:
I dislike the practice of using 'foo'. In tests, when data is needed but the specific value is not important, it's common to use 'dummy'. If the value is used during the test, I use something like 'test-value'.

Reply:
@elopio what about the rest of the functions in the same class? I see them using foo, should we do anything about them?

Reply:
not on this PR, but it would be nice to update them too in a separate one.
Not a big deal, and not everybody will agree with my style, but I find it nice to do at least in the new code.